### ISSUE TITLE
Atomic mass

### ISSUE THREAD

**Issue by luyaozou** [jevandezande/zmatrix#1]
Hello, 
I was wondering shouldn't we use atomic mass of each isotope, instead of the averaged atomic mass (eg. 12C=12 amu instead of C=12.011)? Is it because the change of mole definition in 2018 ?

**Comment by jevandezande** [comment]
Yes, the true atomic mass would be better than the averaged atomic mass. Feel free to submit a PR.

It should be noted that this is a very poor implementation of a zmatrix (written before I understood functional programming). While it works, the code is ugly and I'd recommend writing your own.
